src/main/java/org/c4sg/controller/ProjectController.java

```diff
@@ -37,7 +37,7 @@
 @RestController
 @RequestMapping("/api/projects")
 @Api(description = "Operations about Projects", tags = "project")
-public class ProjectController extends GenericController {
```

Comment:
GenericController contains only individual exceptionHandlers. I configured the application to use GlobalExceptionHandler approach using @RestControllerAdvice. 
      Prior to my code change ProjectController was using GenericController exceptionHandler to throw the exception. With my new approach we don't need inheritance here
     Let me know if its other way around. Thanks